src/main/java/com/sprint/mission/discodeit/controller/AuthController.java

```diff
@@ -20,12 +33,57 @@
 public class AuthController implements AuthApiDocs {
 
   private final AuthService authService;
+  private final UserMapper mapper;
+  private final UserService userService;
+  private final PersistentTokenRepository tokenRepository;
+  private final UserOnlineStatusService statusService;
 
-  @Override
-  @PostMapping("/login")
-  public ResponseEntity<UserResponseDto> userLogin(@Valid @RequestBody LoginDto loginDto) {
-    log.debug("[LOGIN REQUEST] : [USERNAME:{}]", loginDto.username());
-    UserResponseDto user = authService.login(loginDto.username(), loginDto.password());
-    return ResponseEntity.ok(user);
+  @GetMapping("/me")
+  public ResponseEntity<UserResponseDto> me(Authentication authentication) {
+    if (authentication == null
+        || !(authentication.getPrincipal() instanceof DiscodeitUserDetails discodeitUser)) {
+      return ResponseEntity.status(401).build();
+    }
```

Comment:
```suggestion
  @GetMapping("/me")
  public ResponseEntity<UserResponseDto> me(@AuthenticationPrincipal DiscodeitUserDetails discodeitUser) {
    if (discodeitUser == null) {
      return ResponseEntity.status(401).build();
    }
```
AuthenticationPrincipal 어노테이션을 사용하면 principal을 가져올 수 있어요~